### PR TITLE
add support for recipes and update to 1.16.1

### PR DESCRIPTION
**Pull request by user11681**

### CODE REVIEW

File: src/main/java/net/devtech/arrp/json/recipe/JStonecuttingRecipe.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package net.devtech.arrp.json.recipe;
+
+public class JStonecuttingRecipe extends JRecipe {
+    private final JIngredient ingredient;
+    private final String result;
+    private final int count;
```
Comment (Devan-Kerman) [inline src/main/java/net/devtech/arrp/json/recipe/JStonecuttingRecipe.java:6]: is the count field optional?, if so make it an Integer so GSON will omit it

Comment (Devan-Kerman) [reply src/main/java/net/devtech/arrp/json/recipe/JStonecuttingRecipe.java:6]: Same goes for all the other primitives

Comment (user11681) [reply src/main/java/net/devtech/arrp/json/recipe/JStonecuttingRecipe.java:6]: from the wiki:
```
Unlike the count field in shaped and shapeless crafting recipes, this count field here is required. 
```

Comment (Devan-Kerman) [reply src/main/java/net/devtech/arrp/json/recipe/JStonecuttingRecipe.java:6]: oh ok, that's fine then

**Comment by user11681** [comment]
I am unsure about how to resolve the conflicts. Do I have to revert all changes to the listed files?

**Pull request merged**